src/components/svg-canvas-graph.tsx

```diff
@@ -81,6 +91,20 @@ const SvgCanvas = () => {
                 x: ((offset.x - x) * svgViewBoxZoom) / 100,
                 y: ((offset.y - y) * svgViewBoxZoom) / 100,
             });
+        } else if (mode === 'select') {
+            if (selectStart.x != 0 && selectStart.y != 0) {
+                // console.log(
+                //     'Move',
+                //     graph.current.getNodeAttributes(node).x + x,
+                //     graph.current.getNodeAttributes(node).y + y
+                // );
+                dispatch(
```

Comment:
This could be removed without affecting the feature :)

Reply:
No, we won't get the pos of mouse when the mouse is on the nodes.

Reply:
https://github.com/railmapgen/rmp/assets/59787082/694d767e-4974-4558-88d1-fdfaa3ffc093

Reply:
Yes, you're right. Mouse-moving events will be blocked by `e.stopPropagation();` on line 74. I checked the code and looks like if we pop the event up to `handleBackgroundMove` in `svg-wrapper.tsx`, `setSelectMoving` is not required anymore and nothing will break. Correct me if I'm wrong :)

Reply:
Yes, that's right. Now this line can be deleted.